scientific_library/tvb/simulator/monitors.py

```diff
@@ -1026,3 +1026,96 @@ def record(self, step, state):
 
     def sample(self, step, state):
         raise NotImplementedError
+
+
+
+class Ca(Monitor):
```

Comment:
Something more descriptive is required, as this will be imported into a large notebook namespace..